src/routes/publish.ts

```diff
@@ -87,8 +87,8 @@ publish.get('/:datasetId/delete', fetchDataset(Include.Meta), deleteDraft);
 publish.post('/:datasetId/delete', fetchDataset(Include.Meta), deleteDraft);
 
 /* Cube Preview */
-publish.get('/:datasetId/cube-preview', fetchDataset(), cubePreview);
-publish.get('/:datasetId/cube-preview/download', fetchDataset(), downloadDataset);
+publish.get('/:datasetId/cube-preview', fetchDataset(Include.All), cubePreview);
+publish.get('/:datasetId/download', fetchDataset(Include.All), downloadDataset);
```

Comment:
`Include.All` should not be needed here as it's now part of the `cubePreview` controller:

https://github.com/Marvell-Consulting/statswales-frontend/blob/d4b37f332e52a6a3c4cfca550968b9118d6fbb97/src/controllers/publish.ts#L504

`downloadDataset` doesn't use the dataset at all:
https://github.com/Marvell-Consulting/statswales-frontend/blob/d4b37f332e52a6a3c4cfca550968b9118d6fbb97/src/controllers/publish.ts#L562

I've just spent ages removing these calls as they can add 500ms+ every time they're hit 😄